app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class ApiController < ApplicationController
+      include Api::Concerns::ActAsApiRequest
+      include DeviseTokenAuth::Concerns::SetUserByToken
+
+      respond_to :json
+
+      rescue_from ActiveRecord::RecordInvalid, with: :render_record_invalid
+      rescue_from ActiveRecord::RecordNotFound, with: :render_not_found
+
```

Comment:
I think you can also add a `rescue_from Exception` for all the other exceptions that are not contemplated in these two.